docs/docs-components/MarkdownPage.js

```diff
@@ -186,36 +194,40 @@ const components = {
     width?: number,
     height?: number,
     noPadding?: boolean,
-  |}) => (
-    <Box>
-      <Box padding={noPadding ? 0 : 8} rounding={2} borderStyle="sm" height="250px">
-        <Box
-          position="relative"
-          width="100%"
-          height="100%"
-          display="flex"
-          alignItems="center"
-          justifyContent="center"
-        >
-          <Image
-            src={src}
-            alt={alt}
-            width={width || '100%'}
-            height={height || '100%'}
-            layout={width || height ? 'fixed' : 'fill'}
-            objectFit="contain"
-          />
+  |}) => {
+    const layout = width || height ? 'fixed' : 'fill';
+
+    return (
+      <Box>
+        <Box padding={noPadding ? 0 : 8} rounding={2} borderStyle="sm" height="250px">
+          <Box
+            position="relative"
+            width="100%"
+            height="100%"
+            display="flex"
+            alignItems="center"
+            justifyContent="center"
+          >
+            <Image
+              src={src}
+              alt={alt}
+              width={layout === 'fill' ? undefined : width || '100%'}
```

Comment:
linter was complaining about width and height being defined with `layout="fill"`